contracts/exchange-libs/contracts/src/LibExchangeSelectors.sol

```diff
@@ -22,9 +22,6 @@ pragma solidity ^0.5.9;
 contract LibExchangeSelectors {
```

Comment:
I think we can probably ditch this file, right?

Reply:
Removed it entirely from the Exchange contracts. Coordinator and DevUtils still use it quite heavily, so I added it to the backlog.